malaria-ios/ViewControllers/UserProfilePillTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import DoneToolbarSwift
+
+// MARK: - Table View Cell
+
+class UserProfilePillTableViewCell: UITableViewCell {
+  
+  @IBOutlet weak var pillName: UILabel!
+  @IBOutlet weak var quantityTextField: UITextField!
+  
+  weak var delegate : SaveRemainingPillsProtocol?
+  
+  var toolBar: ToolbarWithDone!
+  
+  func updateCellWithParameters(delegate: SaveRemainingPillsProtocol,
```

Comment:
I'm trying to keep lines under 80 characters, but I'm not sure if it's necessary anymore (I will check the coding styles again).